src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -14,6 +15,30 @@
 
 namespace Dynamo.Configuration
 {
+    static class ExtensionMethods
+    {
+        /// <summary>
+        /// Copy Properties from a PreferenceSettings instance to another
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="destination"></param>
+        internal static void CopyProperties(this PreferenceSettings source, PreferenceSettings destination)
+        {
+            // Iterate the Properties of the destination instance and populate them from their source counterparts  
+            PropertyInfo[] destinationProperties = destination.GetType().GetProperties();
```

Comment:
I'll mention this again, `var` makes refactoring much easier and it makes the code easier to read for others because it reduces the line noise.

please use var where possible, if not for those reasons, because it matches the existing style of the rest of the source as well.

Reply:
Deone.